ci/doctests.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+echo "inside $0"
+
+
+source activate pandas
+cd "$TRAVIS_BUILD_DIR"
+
+RET=0
+
+if [ "$DOCTEST" ]; then
+
+    echo "Running doctests"
+
+    # pytest --doctest-modules --ignore=pandas/tests -v  pandas/core/frame.py
+
+    # if [ $? -ne "0" ]; then
+    #     RET=1
+    # fi
+
+
+    # # top-level reshaping functions
```

Comment:
are these just not-working tests?

Reply:
Well, they are failing, and don't want to start fixing all of them here. Will remove or either skip the failing ones